content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -0,0 +1,151 @@
+---
+reviewers:
+- mikedanese
+title: Install and Set Up kubectl on macOS
+content_type: task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: Install kubectl on macOS
+---
+
+## {{% heading "prerequisites" %}}
+
+You must use a kubectl version that is within one minor version difference of your cluster.
+For example, a v1.2 client should work with v1.1, v1.2, and v1.3 master.
+Using the latest version of kubectl helps avoid unforeseen issues.
+
+## Install kubectl on macOS
+
+The following methods exist for installing kubectl on macOS:
+
+- [Install kubectl binary with curl on macOS](#install-kubectl-binary-with-curl-on-macos)
+- [Install with Homebrew on macOS](#install-with-homebrew-on-macos)
+- [Install with Macports on macOS](#install-with-macports-on-macos)
+- [Install on Linux as part of the Google Cloud SDK](#install-on-linux-as-part-of-the-google-cloud-sdk)
+
+### Install kubectl binary with curl on macOS
+
+1. Download the latest release:
+
+   ```bash
+   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl"
+   ```
+
+   {{< note >}}
+   To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
+
+   For example, to download version {{< param "fullversion" >}} on macOS, type:
+
+   ```bash
+   curl -LO https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/darwin/amd64/kubectl
+   ```
+
+   {{< /note >}}
+
+1. Validate the binary (optional)
+
+   Download the kubectl checksum file:
+
+   ```bash
+   curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl.sha256"
+   ```
+
+   Validate the kubectl binary against the checksum file:
+
+   ```bash
+   echo "$(<kubectl.sha256)  kubectl" | shasum -a 256 --check
+   ```
+
+   If valid, the output is:
+
+   ```bash
```

Comment:
use 'none' or 'console' is better.

Reply:
fixed [here](https://github.com/kubernetes/website/pull/26682/commits/fb2b33ef3463546abb35270108f030317355ce12)